cms/templates/widgets/footer.html

```diff
@@ -27,6 +28,11 @@
                 <a data-rel="edx.org" href="${marketing_link('PRIVACY')}">${_("Privacy Policy")}</a>
               </li>
             % endif
+            % if waffle().is_enabled(ENABLE_ACCESSIBILITY_POLICY_PAGE):
+              <li class="nav-item nav-peripheral-aar">
+                <a data-rel="edx.org" href="/accessibility">${_("Accessibility Accomodation Request")}</a>
```

Comment:
I think we can use `reverse` to get the URL by name (i.e. "accessibility") in case the path changes, e.g. https://github.com/edx/edx-platform/blob/8bb56fa44822321b6fe16a0cd7825d882a014deb/themes/stanford-style/lms/templates/footer.html#L15